reana_server/config.py

```diff
@@ -61,6 +61,9 @@
 DASK_ENABLED = strtobool(os.getenv("DASK_ENABLED", "true"))
 """Whether Dask is enabled in the cluster or not"""
 
+DASK_AUTOSCALER_ENABLED = strtobool(os.getenv("DASK_AUTOSCALER_ENABLED", "true"))
```

Comment:
See the comment about "true" vs "True" in https://github.com/reanahub/reana-workflow-controller/pull/604#discussion_r1832813431